### ISSUE TITLE
Work with groups

### ISSUE THREAD

**Issue by pepeloper** [jaxbot/hangouts-bot#13]
This may be a feature request but there is a way to make it work with hangouts groups?

As default it doesn't hears when a user type to a group.

**Comment by privrax** [comment]
yeah according to issue #3 's answers it ins't possible to add the feature to this repo :(
